src/app/services/views/ServiceDetailView.vue

```diff
@@ -39,125 +39,124 @@
-                            :fields="{
-                              name: { description: 'filter by name or parts of a name' },
-                              protocol: { description: 'filter by “kuma.io/protocol” value' },
-                              tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
-                              zone: { description: 'filter by “kuma.io/zone” value' },
-                            }"
-                            @fields-change="(val) => route.update({
-                              query: val.query,
-                              s: val.query.length > 0 ? JSON.stringify(val.fields) : ''
-                            })"
-                          />
-
-                          <KSelect
-                            v-if="gateways"
-                            label="Type"
-                            :overlay-label="true"
-                            :items="[
-                              {
-                                label: 'All',
-                                value: 'all'
-                              },
-                              {
-                                label: 'Builtin',
-                                value: 'builtin'
-                              },
-                              {
-                                label: 'Delegated',
-                                value: 'delegated'
-                              }
-                            ].map(item => ({
-                              ...item,
-                              selected: item.value === props.gatewayType
-                            }))"
-                            appearance="select"
-                            @selected="(item: SelectItem) => route.update({
-                              gatewayType: String(item.value),
-                            })"
-                          >
-                            <template #item-template="{ item: value }">
-                              {{ value.label }}
-                            </template>
-                          </KSelect>
-                        </template>
-                      </DataPlaneList>
-                    </template>
-                  </KCard>
-                </template>
-              </DataSource>
-            </template>
-          </TabsWidget>
-        </DataSource>
+                <KCard>
+                  <template #body>
+                    <DataPlaneList
+                      data-testid="data-plane-collection"
+                      class="data-plane-collection"
+                      :page-number="props.page"
+                      :page-size="props.size"
+                      :total="dataplanesData?.total"
+                      :items="dataplanesData?.items"
+                      :error="dataplanesError"
+                      :gateways="gateways"
+                      @change="({page, size}) => {
+                        // @TODO: Should we remove s: undefined?
```

Comment:
What is this TODO for? Should it be removed?

Reply:
I don’t know.

Reply:
Should we just remove it then?

Reply:
@johncowen Fine by you? We have this in three places. Could we just do `@change="route.update"` like in other places?

Reply:
Yeah sure fine by me to remove this comment.